### ISSUE TITLE
IoU funcs in voxel.py and point.py have the same name

### ISSUE THREAD

**Issue by justanhduc** [NVIDIAGameWorks/kaolin#11]
as the title indicates.

**Comment by krrish94** [comment]
Good point. While for now you can get around this by explicitly accessing `metrics.voxel.iou` and `metrics.point.iou`, we will look into renaming/overloading this module real soon.
